.github/PULL_REQUEST_TEMPLATE/docs.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Docs
+about: Updates documentation, including specifications.
+---
+
+## Description
+
+<!-- Add a description of the changes that this PR introduces and the files that
+are the most critical to review. -->
+
+closes: #XXXX
```

Comment:
```suggestion
Closes: #XXXX
```

Reply:
and in other files as well.

Reply:
Let's also move the comment below `Closes` - this line should be the first line right below Description.

Reply:
will do, thanks!